src/lib/warn-upload-overwrite.js

```diff
@@ -127,9 +127,13 @@ const preUploadByXml = async (db, docId, localXml) => {
     const buffer = await db.getAttachment(docId, 'xml');
     remoteXml = buffer.toString('utf8');
   } catch (e) {
-    // continue regardless of error
-    log.trace('Trying to fetch remote xml', e);
-    throw e;
+    if (e.status === 404) {
+      return Promise.resolve();
```

Comment:
This could use a comment... something along the lines of...

```suggestion
      // The form doesn't exist on the server so we know we're not overwriting anything
      return Promise.resolve();
```